static/js/apps/nl_interface/query_search.tsx

```diff
@@ -92,6 +101,66 @@ export function QuerySearch(props: QuerySearchPropType): JSX.Element {
             <span className="material-icons">history</span>
           </Button>
         </div>
+        <div className="nl-options-row">
+          <div className="nl-options-label">Detection:</div>
+          <div className="nl-options-input-radio">
+            <FormGroup>
+              <Label>
+                <Input
+                  checked={!props.useLLM}
+                  id="nl-heuristics"
+                  type="radio"
+                  value={0}
+                  onChange={(e) => {
```

Comment:
nit: e is unused so don't need it. Here and the other radio buttons

Reply:
Removed.